content/en/blog/2022/get-started-ambient/index.md

```diff
@@ -17,11 +17,11 @@ Ambient mesh is [a new data plane mode for Istio introduced today](/blog/2022/in
 $ kind create cluster --config=- <<EOF
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-name: ambient
-nodes:
-- role: control-plane
-- role: worker
-- role: worker
+  name: ambient
```

Comment:
This indentation is wrong and it does not work. name and nodes should be at the same level as apiVersion?

Reply:
Not sure. I read the k8s doc yesterday:

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

The YAML of a deployment looks like:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  labels:
    app: nginx
spec:
  replicas: 3
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
```

Reply:
this's a cluster, not deployment

Reply:
sourcode is here: https://github.com/kubernetes-sigs/kind/blob/d53225750af998c6218b5c8c730b8b3ca0e2d472/pkg/apis/config/v1alpha4/types.go#L20

Reply:
Understood. This block is changed back. 😸